web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -893,7 +893,12 @@
     "system/searchStats": "Search statistics",
     "system/searchStats/enable": "Enable",
     "system/searchStats/enable-help": "If set, statistics on searches will be collected and stored in the database.",
-    "system/selectionmanager": "Metadata Search Results",
+    "system/security": "Security",
+    "system/security/passwordEnforcement/maxLength": "Password max. length",
+    "system/security/passwordEnforcement/minLength": "Password min. length",
+    "system/security/passwordEnforcement/usePattern": "Password restrictions",
+    "system/security/passwordEnforcement/usePattern-help": "Require that the password must contain at least 1 uppercase, 1 lowercase, 1 number and 1 symbol",
```

Comment:
It would be nice if the catalog admin could change the usePattern-help string from the admin interface. Not all catalog admins have access to the code. Also the error message in core/src/main/resources/org/fao/geonet/api/Messages.properties. At present they can change the regex but can't update the messages to match it.

Reply:
This may be feasible using the API (not yet any interface) to override translations ? See https://github.com/geonetwork/core-geonetwork/pull/4756